curriculum/challenges/english/blocks/workshop-parent-teacher-conference-form/68eb29fa879c7deef9806a04.md

```diff
@@ -0,0 +1,322 @@
+---
+id: 68eb29fa879c7deef9806a04
+title: Step 28
+challengeType: 0
+dashedName: step-28
+---
+
+# --description--
+
+To ensure the text entered into inputs and textareas, as well as their placeholders, is visible, set their color.
+
+Select `input, input::placeholder, textarea` and set `color` to `whitesmoke`.
+
+# --hints--
+
+You should target `input`, `input::placeholder`, `textarea`.
+
+```js
+assert(new __helpers.CSSHelp(document).getStyle('input, input::placeholder, textarea'));
+```
+
+Set the `color` property to `whitesmoke`.
+
+```js
+const style = new __helpers.CSSHelp(document).getStyle('input, input::placeholder, textarea');
+assert.strictEqual(style?.backgroundColor, 'whitesmoke');
```

Comment:
this test is failing, you asked to set `color` not `backgroundColor` in the hint